client/state/selectors/is-site-vip.js

```diff
@@ -0,0 +1,22 @@
+/**
```

Comment:
Maybe it would make sense to move the new `isSiteVip` selector and its tests to a separate PR?